hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/clustering/ClusteringCommitSink.java

```diff
@@ -141,6 +141,15 @@ private void commitIfNecessary(String instant, List<ClusteringCommitEvent> event
     }
   }
 
+  private boolean containFailedEvent(List<ClusteringCommitEvent> events) {
+    return events.stream().anyMatch(event -> {
+      if (event.isFailed()) {
+        LOG.warn("Failed clustering event from task " + event.getTaskID() + ", will rollback instant " + event.getInstant());
+      }
```

Comment:
Is the log helping by pointing out the task id? If not, just log the message directly in line 124.

Reply:
yes, log failed taskid to help finding error that fail the event.